setup.py

```diff
@@ -47,53 +46,50 @@ def read(fname):
     'singledispatchmethod; python_version < "3.8.0"',
     'dataclasses; python_version < "3.7.0"',
     'PyYAML>=5.1',
-    'orjson>=3.6.1',    # this is a faster json lib support inf/nan and datetime
-                        # v3.6.1 is the last version supports Python 3.6
+    'orjson>=3.6.1',  # this is a faster json lib support inf/nan and datetime
+    # v3.6.1 is the last version supports Python 3.6
```

Comment:
I also don't like this line of fix. The line of comment was part of the line above.